public/app/screens/main/sections/settings/user-form/user-form.jsx

```diff
@@ -0,0 +1,49 @@
+var React = require('react');
+var Reflux = require('reflux');
+var styleMixin = require('mixins/style-mixin');
+var t = require('tcomb-form');
```

Comment:
Because saying tcombForm relates more to some concrete form, but tForms more general. 
However I don't mind about your proposition.